draft-ietf-rats-architecture.md

```diff
@@ -1188,64 +1188,80 @@ in terms of time(RX)-time(RG), then the Relying Party can check
 
 ## Example 3: Timestamp-based Background-Check Model Example
 
+The following example illustrates a hypothetical Background-Check Model solution that uses timestamps and requires roughly synchronized clocks between the Attester, Verifier, and Relying Party.
```

Comment:
Isn't this part covered by PR 124 now?   can remove from this one or abandon this one if other PRs will cover it

Reply:
That is probably an artifact.